.github/workflows/refresh-block-time.yml

```
@@ -0,0 +1,41 @@
+name: Refresh Block Times
+
+on:
+  schedule:
+    # Run once a month
+    - cron: '0 12 1 * *'
+
+jobs:
+  refresh-block-times:
+    runs-on: ubuntu-latest
+    env:
+      branchName: action-refresh-block-time
+    steps:
+      - name: Clone @api3/chains
+        uses: actions/checkout@v3
```

I had to do something with `actions/checkout` yesterday and saw that v4 is out now
```suggestion
        uses: actions/checkout@v4
```